lib/puppet/provider/mount/parsed.rb

```diff
@@ -253,16 +275,17 @@ def self.prefetch(resources = nil)
   end
 
   def self.mountinstances
-    # XXX: Will not work for mount points that have spaces in path (does fstab support this anyways?)
     regex = case Facter.value(:osfamily)
             when 'Darwin'
               %r{ on (?:/private/var/automount)?(\S*)}
             when 'Solaris', 'HP-UX'
               %r{^(\S*) on }
             when 'AIX'
               %r{^(?:\S*\s+\S+\s+)(\S+)}
+            when %r{FreeBSD|NetBSD}i
+              %r{ on (.*) \(}
```

Comment:
this was done based on the fixtures, I also checked outputs with a real FreeBSD box, I don't have a NetBSD though.